diff -up shotcut-19.12.16/src/src.pro.omv~ shotcut-19.12.16/src/src.pro
--- shotcut-19.12.16/src/src.pro.omv~	2019-12-24 00:01:25.123225780 +0100
+++ shotcut-19.12.16/src/src.pro	2019-12-24 00:02:34.077222435 +0100
@@ -1,6 +1,6 @@
 CONFIG   += link_prl
 
-QT       += widgets opengl xml network printsupport qml quick sql webkitwidgets
+QT       += widgets opengl xml network printsupport qml quick sql testlib webkitwidgets
 QT       += multimedia websockets quickwidgets
 QT       += qml-private core-private quick-private gui-private
 
