diff -up shotcut-20.04.12/src/widgets/scopes/videohistogramscopewidget.cpp.omv~ shotcut-20.04.12/src/widgets/scopes/videohistogramscopewidget.cpp
--- shotcut-20.04.12/src/widgets/scopes/videohistogramscopewidget.cpp.omv~	2020-04-13 15:07:15.235962057 +0200
+++ shotcut-20.04.12/src/widgets/scopes/videohistogramscopewidget.cpp	2020-04-13 15:07:20.150017514 +0200
@@ -19,6 +19,7 @@
 #include <Logger.h>
 #include <QMouseEvent>
 #include <QPainter>
+#include <QPainterPath>
 #include <QToolTip>
 
 const qreal IRE0 = 16;
